konfera/event/views.py

```diff
@@ -224,34 +227,115 @@ def event_public_tickets(request, slug):
     return render(request=request, template_name='konfera/event/public_tickets.html', context=context)
 
 
-def event_order_detail(request, order_uuid):
-    order = get_object_or_404(Order, uuid=order_uuid)
+def event_about_us(request, slug):
+    event = get_object_or_404(Event.objects.published(), slug=slug)
     context = dict()
+    update_event_context(event, context)
+
+    if not event.organizer:
+        raise Http404(_('Organizer has not been set for event %s' % event.title))
+
+    context['organizer'] = event.organizer
+
+    return render(request=request, template_name='konfera/event/event_organizer.html', context=context)
+
+
+class EventOrderDetailView(DetailView):
+    model = Order
+    template_name = 'konfera/order_detail.html'
+    slug_field = 'uuid'
+    slug_url_kwarg = 'order_uuid'
 
-    if order.event:
-        update_event_context(order.event, context, show_sponsors=False)
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data()
+        context['order'] = self.object
+        context['allow_receipt_edit'] = True
+        context['allow_pdf_storage'] = True
+
+        if self.object.status != Order.AWAITING:
+            context['allow_receipt_edit'] = False
+
+        if self.object.event:
+            update_event_context(self.object.event, context, show_sponsors=False)
+
+        update_order_status_context(self.object.status, context)
+
+        return context
+
+
+class EventOrderDetailFormView(ModelFormMixin, EventOrderDetailView):
+    form_class = ReceiptForm
+
+    def dispatch(self, *args, **kwargs):
+        self.object = self.get_object()
 
-    if order.status == Order.AWAITING:
-        context['form'] = form = ReceiptForm(request.POST or None, instance=order.receipt_of)
+        if self.object.status != Order.AWAITING:
+            messages.error(self.request, _('You can not edit order. Please contact support.'))
```

Comment:
*You cannot edit your order. Please contact the support.*

sounds better in my opinion